script/release

```diff
@@ -0,0 +1,436 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+#/ Usage: GH_RELEASE_TOKEN=your-amazing-secure-token release [--dry-run] <version>
+#/
+#/ Publish a backup-utils release:
+#/ * Updates the package changelog
+#/ * Bumps the backup-utils version if required
+#/ * Creates the release pull request
+#/ * Merges the release pull request
+#/ * Creates the release draft
+#/ * Tags the release
+#/ * Builds the release assets and uploads them
+#/
+#/ Notes:
+#/ * Needs GH_RELEASE_TOKEN available in the environment.
+#/ * Export GH_OWNER, GH_AUTHOR and GH_REPO if you want to tweak the build
+#/   changelog or use a different  owner/repo
+#/ * Only pull requests labeled with bug, feature or enhancement will show up in the
+#/   release page and the changelog.
+#/
+require 'json'
+require 'net/http'
+require 'time'
+require 'erb'
+require 'English'
+
+API_HOST = ENV['GH_HOST'] || 'api.github.com'
+API_PORT = 443
+GH_REPO = ENV['GH_REPO'] || 'backup-utils'
+GH_OWNER = ENV['GH_OWNER'] || 'github'
+GH_AUTHOR = ENV['GH_AUTHOR'] || 'Sergio Rubio <rubiojr@github.com>'
```

Comment:
I'd make this env var mandatory, I don't want to be credited for the work others did 😜.